sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptionsFactory.java

```diff
@@ -1085,33 +1107,72 @@ private static void throwForTypeMismatches(List<TypeMismatch> mismatches) {
     }
     throwForGettersWithInconsistentJsonIgnore(incompletelyIgnoredGetters);
     throwForSettersWithJsonIgnore(ignoredSetters);
+  }
 
+  /**
+   * Validates that every bean property of any interface derived from {@link PipelineOptions}
```

Comment:
Validates that every bean property of the given interface must have both a getter and setter.